profiles/schema.py

```diff
@@ -0,0 +1,107 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import DivisionOfInterest, Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+    vocabulary = graphene.String()
+    code = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+    name = graphene.String()
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+        exclude = ("id", "user")
```

Comment:
we have to change our approach with the fields. let's specify which fields are available, instead of specifying which fields are excluded, since right now we expose too much:
![image](https://user-images.githubusercontent.com/24206160/63914461-f2fc1a00-ca3b-11e9-80a6-58892036a53c.png)

Reply:
```python
fields = (
    "nickname",
    "image",
    "email",
    "phone",
    "language",
    "contact_method",
    "concepts_of_interest",
    "divisions_of_interest"
)
```